src/app/globals.css

```diff
@@ -2,8 +2,27 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url("https://fonts.googleapis.com/css2?family=Heebo:wght@100..900&display=swap");
+
+@font-face {
+  font-family: "BaruSans";
```

Comment:
It seems like there was a discussion about switching to Heebo from Baru. Do we need to define the BaruSans font here still?

Reply:
I don't know if Justin still wants to keep it or not